packages/shared/getComponentName.js

```diff
@@ -63,13 +64,23 @@ function getComponentName(type: mixed): string | null {
         return 'Context.Consumer';
       case REACT_PROVIDER_TYPE:
         return 'Context.Provider';
-      case REACT_FORWARD_REF_TYPE:
+      case REACT_FORWARD_REF_TYPE: {
         const renderFn = (type.render: any);
         const functionName = renderFn.displayName || renderFn.name || '';
         return (
           (type: any).displayName ||
           (functionName !== '' ? `ForwardRef(${functionName})` : 'ForwardRef')
```

Comment:
should we be consistent?

Reply:
Maybe.. I figured this one would be common and might be annoying to see it everywhere. Or maybe not?

Reply:
I guess it could be helpful to see that a pure component re-rendered